HPXMLtoOpenStudio/resources/hotwater_appliances.rb

```diff
@@ -113,16 +116,31 @@ def self.apply(model, weather, living_space,
     end
 
     # Refrigerator
-    if not refrigerator.nil?
-      rf_annual_kwh, rf_frac_sens, rf_frac_lat = calc_refrigerator_energy(refrigerator, rf_space.nil?)
+    if not refrigerators.empty?
       fridge_name = Constants.ObjectNameRefrigerator
       fridge_weekday_sch = '0.040, 0.039, 0.038, 0.037, 0.036, 0.036, 0.038, 0.040, 0.041, 0.041, 0.040, 0.040, 0.042, 0.042, 0.042, 0.041, 0.044, 0.048, 0.050, 0.048, 0.047, 0.046, 0.044, 0.041'
       fridge_monthly_sch = '0.837, 0.835, 1.084, 1.084, 1.084, 1.096, 1.096, 1.096, 1.096, 0.931, 0.925, 0.837'
       fridge_schedule = MonthWeekdayWeekendSchedule.new(model, fridge_name, fridge_weekday_sch, fridge_weekday_sch, fridge_monthly_sch, 1.0, 1.0, true, true, Constants.ScheduleTypeLimitsFraction)
-      fridge_design_level = fridge_schedule.calcDesignLevelFromDailykWh(rf_annual_kwh / 365.0)
+      refrigerators.zip(rf_spaces).each do |refrigerator, rf_space|
+        rf_annual_kwh, rf_frac_sens, rf_frac_lat = calc_refrigerator_energy(refrigerator, rf_space.nil?)
+        fridge_design_level = fridge_schedule.calcDesignLevelFromDailykWh(rf_annual_kwh / 365.0)
 
-      rf_space = living_space if rf_space.nil?
-      add_electric_equipment(model, fridge_name, rf_space, fridge_design_level, rf_frac_sens, rf_frac_lat, fridge_schedule.schedule)
+        rf_space = living_space if rf_space.nil?
+        add_electric_equipment(model, fridge_name, rf_space, fridge_design_level, rf_frac_sens, rf_frac_lat, fridge_schedule.schedule)
+      end
+    end
+
+    # Freezer
+    if not freezer.nil?
+      fz_annual_kwh, fz_frac_sens, fz_frac_lat = calc_refrigerator_energy(freezer, fz_space.nil?)
```

Comment:
Is it intentionally calling calc_refrigerator_energy? If so, rename to calc_refrigerator_or_freezer_energy.